### PR TITLE
The Tailorening

### PR DESCRIPTION
**Pull request by Polyzonial**
## About The Pull Request
Adds new sewing recipes and leather vests to skincrafting.

## Why It's Good For The Game
New stuff, tailoring is a more valuable skill now. 
Open to suggestions.

### CODE REVIEW

**Comment by lovegreenstuff** [comment]
you forgot to rename the leather vest crafting recipe so it overrides saddle

File: code/modules/roguetown/roguecrafting/sewing.dm
Comment (Wintermote) [inline code/modules/roguetown/roguecrafting/sewing.dm:1]: >datum/crafting_recipe/roguetown/sewing/headband
	name = "headband"
	result = list(/obj/item/clothing/head/roguetown/headband)
	reqs = list(/obj/item/natural/cloth = 1)
	craftdiff = 0

>/datum/crafting_recipe/roguetown/sewing/redheadband
	name = "red headband"
	result = list(/obj/item/clothing/head/roguetown/headband)
	reqs = list(/obj/item/natural/cloth = 1)
	craftdiff = 0

Do these not make the exact same item?

Also, my experience with trying to craft a gambeson is spending 40 minutes training sewing to level 3 and still having a 0% chance, and this after having collected probably upwards of 60-80 fibers; and now your PR is putting it at an even higher skill level. 

Basically, in my experience it is easier to get full plate armor from a smith than it is to make a gambeson.

**Comment by thefirstjeb** [comment]
why not add a tailor role already?

**Comment by Real-MAGNUM** [comment]
Merging. Feel free to re-add the tailor role.

**Comment by Hmeister-real** [comment]
> why not add a tailor role already?

already is, it's called a women
